src/test/java/org/testeditor/fixture/web/io/FileReaderTest.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2017 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+package org.testeditor.fixture.web.io;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileReaderTest {
+
+    static final String lineSeparator = System.getProperty("line.separator");
+        
+    String result = "firefox-cap1 = \"abcdüöß\"" + lineSeparator  
+            + "firefox-cap2 = 8" + lineSeparator  
+            + "firefox-cap3 = true";
+
+    @Test
+    public void testReadFile() throws Exception {
+        FileReader reader = new FileReader("test.txt");
```

Comment:
test.txt is a bit generic, how about utf8EncodedTextWUmlaut.txt or something close to that?